### ISSUE TITLE
[vSphere] Free Bytes on Filesystem in Web Consoles node overview is wrong (negative and huge)

### ISSUE THREAD

**Issue by jomeier** [openshift/okd#286]
Hi,

OpenShift Version
4.5.0-0.okd-2020-07-29-070316

In Compute->Nodes->Node Overview in Filesystem the calculated number of free Bytes is wrong (negative big number):

![filesystem](https://user-images.githubusercontent.com/12024238/89002363-23f46700-d2fd-11ea-9a9c-c8b2573f318d.PNG)
![nodes](https://user-images.githubusercontent.com/12024238/89002156-aaf50f80-d2fc-11ea-92ae-bf17195c7353.PNG)
![nodew1](https://user-images.githubusercontent.com/12024238/89002245-e4c61600-d2fc-11ea-9286-3da945be8277.PNG)

It seems to be a display bug because one level above in the list of all nodes the free bytes seem to be correct.

Greetings,

Josef

**Comment by vrutkovs** [comment]
Right, that's caused by node-exporter reporting all available mounts (apparently including containers) and Prometheus query summing all of those. Found a few references to this issue internally, but all of those are closed (as technically monitoring is doing the right thing).

Not sure how to proceed on this

**Comment by jomeier** [comment]
I made several installations (vSphere / Proxmox) without fiddling around with the CMO, vanilla OKD 4.5 GA.

Everywhere this value is negative and huge. It seems to be a bug.

**Comment by TecHunter** [comment]
Same here on Bare Metal. did the install twice and always happened the same. lsblk result :
```

nvme0n1     259:0    0   477G  0 disk
|-nvme0n1p1 259:1    0   384M  0 part /boot
|-nvme0n1p2 259:2    0   127M  0 part /boot/efi
|-nvme0n1p3 259:3    0     1M  0 part
`-nvme0n1p4 259:4    0 476.4G  0 part /sysroot
```

**Comment by vrutkovs** [comment]
Related: https://github.com/openshift/console/pull/7201 (4.6 cherry-pick - https://github.com/openshift/console/pull/7278)

**Comment by vrutkovs** [comment]
This was resolved in 4.6/4.7 stable, please reopen if the issue persists
